### ISSUE TITLE
Missing examples on how to validate conllup and check XPOS to UPOS correspondence

### ISSUE THREAD

**Issue by nljubesi** [reldi-data/reldi-data#2]
There are only four conllup files, so, besides documenting, what the fancy tools can do, it would be very good we had either bash scripts (as for train:dev:test splitting), or at least commands documented that are to be run for validating the format and the XPOS to UPOS correspondence.

On top of that, I suggest you describe in short in the README what the validation actually does. It uses the UD validation script, right?

UPOS to XPOS correspondence I will quite likely further work on and will document accordingly.

**Comment by nljubesi** [comment]
The full documentation of the specific tools is not to be kept in the main readme (to be renamed to README.md, Readme.md is rather unusual), but to be moved to additional readmes. The main readme is to give clear instructions on how to download submodules, further process them, and validate them. Now there is a lot of something there.

**Comment by nljubesi** [comment]
The READMEs of specific datasets have improved. With the validation README of hr500k I even corrected an error (@tsamardzic please check https://github.com/reldi-data/hr500k/commit/17ea8e1cdea30d322e26b17938463b18bf845bc5). There is still a lot to be done, especially on the README.md of this repository. @vstefanec if it is not clear to you how to do it by yourself, I will do it.

**Comment by nljubesi** [comment]
Ah, yes, and resolving one error (https://github.com/reldi-data/hr500k/commit/17ea8e1cdea30d322e26b17938463b18bf845bc5) mostly just makes other errors visible. @vstefanec please check what is with this `NamedEntity=Yes` in the token I have corrected. Is there a systemic issue?

**Comment by vstefanec** [comment]
No, it is not. It's a random error. Corrected with https://github.com/reldi-data/hr500k/commit/b66793ed556c1cd9ca8b05731ccce9a889451a6f.

**Comment by vstefanec** [comment]
@nljubesi , if you could take over the reshuffling of READMEs, that would be great.

**Comment by nljubesi** [comment]
> No, it is not. It's a random error. Corrected with [reldi-data/hr500k@b66793e](https://github.com/reldi-data/hr500k/commit/b66793ed556c1cd9ca8b05731ccce9a889451a6f).

This is SO weird - the token that I find via the report to have an inconsistency between UPOS and XPOS, to be the only one to have an attribute, that is being added automatically, added by mistake. However, I checked all corpora (some grepping) and could not find similar issues.

**Comment by vstefanec** [comment]
@nljubesi , this is merely a coincidence. The error was present in the source file already.

**Comment by tsamardzic** [comment]
> The READMEs of specific datasets have improved. With the validation README of hr500k I even corrected an error (@tsamardzic please check [reldi-data/hr500k@17ea8e1](https://github.com/reldi-data/hr500k/commit/17ea8e1cdea30d322e26b17938463b18bf845bc5)). There is still a lot to be done, especially on the README.md of this repository. @vstefanec if it is not clear to you how to do it by yourself, I will do it.

Checked and left a comment (that the correction is good :)) but I must say I don't understand what it means to correct something with the validation README. 

I have started a new Issue #6 to track the documentation. Lets comment there.

**Comment by nljubesi** [comment]
> > The READMEs of specific datasets have improved. With the validation README of hr500k I even corrected an error (@tsamardzic please check [reldi-data/hr500k@17ea8e1](https://github.com/reldi-data/hr500k/commit/17ea8e1cdea30d322e26b17938463b18bf845bc5)). There is still a lot to be done, especially on the README.md of this repository. @vstefanec if it is not clear to you how to do it by yourself, I will do it.
> 
> Checked and left a comment (that the correction is good :)) but I must say I don't understand what it means to correct something with the validation README.

"With the validation README in place", or "With the help of the validation README". :-) So thanks to the validation README, that might also be auto-generated (git hooks and such), I even realised we had a mistake. What is even more, I found the weird NamedEntity attribute mistake. So reports are great.
